rest/route/project.go

```diff
@@ -1057,11 +1057,11 @@ func (p *GetProjectAliasResultsHandler) Run(ctx context.Context) gimlet.Responde
 	return gimlet.NewJSONResponse(variantTasks)
 }
 
-////////////////////////////////////////////////////////////////////////
+// //////////////////////////////////////////////////////////////////////
```

Comment:
not sure why but my new golang linter is making these changes as i save :(

Reply:
LOL